docs/pages/access-controls/device-trust/guide.mdx

```diff
@@ -45,54 +45,53 @@ protected with Teleport.
   root@(=clusterDefaults.nodeIP=):~#
   ```
 
+<Details type="warning" title="Teleport v13.3.5 and Below">
```

Comment:
Do we need to mention v13 anymore?

Reply:
Our convention has been to remove version warnings for versions below the latest EOL version: https://github.com/gravitational/teleport/blob/master/.github/ISSUE_TEMPLATE/test-plan-docs.md#is-the-docs-site-up-to-date-with-the-new-release